### PR TITLE
fix(starr): Match anime scores between docs and JSON

### PR DESCRIPTION
**Pull request by calvinbui**
# Pull Request

## Purpose

1. Update docs to use CF scores directly from the JSON files.
2. Add anime scores to JSON files `radarr/cf/french-vostfr` and `sonarr/cf/av1` which were taken from the docs.

## Approach

Docs now match JSON files and vice-versa

## Open Questions and Pre-Merge TODOs

N/A

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by yammes08 — APPROVED** [review]
Looks good to me

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
> Looks good to me

See review there's some issues....?

File: docs/json/sonarr/cf/av1.json [hunk 1]
```diff
@@ -9,7 +12,7 @@
       "negate": false,
       "required": true,
       "fields": {
-      "value": "\\bAV1\\b"
+        "value": "\\bAV1\\b"
```
Comment (yammes08) [inline docs/json/sonarr/cf/av1.json:15]:
```suggestion
      "value": "\\bAV1\\b"
```

File: includes/cf/sonarr-anime.md [hunk 2]
```diff
@@ -22,7 +22,7 @@
...
     | [{{ sonarr['cf']['anime-lq-groups']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#anime-lq-groups)                                       |            {{ sonarr['cf']['anime-lq-groups']['trash_scores']['default'] }}             | {{ sonarr['cf']['anime-lq-groups']['trash_id'] }}                    |
-    | [{{ sonarr['cf']['av1']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#av1)                                                               |                                         -10000                                          | {{ sonarr['cf']['av1']['trash_id'] }}                                |
+    | [{{ sonarr['cf']['av1']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#av1)                                                               |                  {{ sonarr['cf']['av1']['trash_scores']['default'] }}                   | {{ sonarr['cf']['av1']['trash_id'] }}                                |
```
Comment (yammes08) [inline includes/cf/sonarr-anime.md:25]:
```suggestion
    | [{{ sonarr['cf']['av1']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#av1)                                                               |                  {{ sonarr['cf']['av1']['trash_scores']['anime-sonarr'] }}                   | {{ sonarr['cf']['av1']['trash_id'] }}                                |
```

Comment (yammes08) [reply includes/cf/sonarr-anime.md:25]: Might need the table formatting tidying here too.

**Review by yammes08 — CHANGES_REQUESTED** [review]
Apologies for the erroneous "looks good to me". Baker's review more comprehensive, see my suggestions.

**Comment by yammes08** [comment]
> > Looks good to me
> 
> See review there's some issues....?

Yeah, there's me thinking I reviewed it properly...obviously not 😄 - suggestions added.

File: includes/cf/sonarr-anime.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (calvinbui) [reply includes/cf/sonarr-anime.md]: my bad. fixed https://github.com/calvinbui/Guides/commit/f194829ba54286fdd363bb86d2a225d29ac28a91

Comment (calvinbui) [reply includes/cf/sonarr-anime.md]: of course. looks okay my computer.

image

File: docs/json/sonarr/cf/av1.json (hunk 1 above)
Comment (calvinbui) [reply docs/json/sonarr/cf/av1.json:15]: @bakerboy448 @yammes08 i'll revert this, but i've noticed a lot of discrepancies in the CFs:

- this change we're commenting on. `value` is a key under `fields` and should be indented.
- formatting an array of objects with like [this](https://github.com/TRaSH-Guides/Guides/blob/0b90b4689f3783d428a562f273729bb9fe1a3859/docs/json/sonarr/cf/dd.json#L9) vs [this](https://github.com/TRaSH-Guides/Guides/blob/0b90b4689f3783d428a562f273729bb9fe1a3859/docs/json/sonarr/cf/ddplus-atmos.json#L9-L10)
- 2 vs 4 space indentation
- incorrect indentation such as https://github.com/TRaSH-Guides/Guides/blob/0b90b4689f3783d428a562f273729bb9fe1a3859/docs/json/sonarr/cf/cr.json#L2-L4

If you like, I could make another PR to resolve these or is this intended?

Comment (bakerboy448) [reply docs/json/sonarr/cf/av1.json]: The jsons should match whatever is expected directly from Starr with only the specific trash fields added

Comment (yammes08) [reply docs/json/sonarr/cf/av1.json]: I think we picked this up after the org merge, that there's a bigger piece of work to do to standardise CF json formatting.

**Review by yammes08 — APPROVED** [review]

**Comment by yammes08** [comment]
@bakerboy448 - can you see which of your changes haven't been addressed? I can never find them..
@nuxencs - sorry for closing the other linked PR. Does this one look ok to you too?

**Review by bakerboy448 — APPROVED** [review]
diff lgtm

File: includes/cf/sonarr-streaming-services-anime.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+    | [{{ sonarr['cf']['nf']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#nf)             |  {{ sonarr['cf']['nf']['trash_scores']['anime-sonarr'] }}  | {{ sonarr['cf']['nf']['trash_id'] }}         |
+    | [{{ sonarr['cf']['amzn']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#amzn)         | {{ sonarr['cf']['amzn']['trash_scores']['anime-sonarr'] }} | {{ sonarr['cf']['amzn']['trash_id'] }}       |
+    | [{{ sonarr['cf']['vrv']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#vrv)           |    {{ sonarr['cf']['vrv']['trash_scores']['default'] }}    | {{ sonarr['cf']['vrv']['trash_id'] }}        |
+    | [{{ sonarr['cf']['funi']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#funi)         |   {{ sonarr['cf']['funi']['trash_scores']['default'] }}    | {{ sonarr['cf']['funi']['trash_id'] }}       |
+    | [{{ sonarr['cf']['french-adn']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#adn)    | {{ sonarr['cf']['funi']['trash_scores']['anime-sonarr'] }} | {{ sonarr['cf']['french-adn']['trash_id'] }} |
```
Comment (nuxencs) [inline includes/cf/sonarr-streaming-services-anime.md:10]:
```suggestion
    | [{{ sonarr['cf']['french-adn']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#adn)    | {{ sonarr['cf']['french-adn']['trash_scores']['anime-sonarr'] }} | {{ sonarr['cf']['french-adn']['trash_id'] }} |
```

**Review by nuxencs — CHANGES_REQUESTED** [review]

**Review by yammes08 — APPROVED** [review]

**Pull request merged**